package.json

```diff
@@ -7,8 +7,8 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "start": "mkdir -p extension && cp manifest.json extension/ && parcel src/github/background.js icons/* src/constants.js src/**/*.html src/**/*.css --dist-dir extension",
-    "build": "mkdir -p extension && cp manifest.json extension/ && parcel build src/github/background.js icons/* src/constants.js src/**/*.html src/**/*.css --dist-dir extension",
+    "start": "parcel watch src/manifest.json --config @parcel/config-webextension --host localhost --no-cache",
```

Comment:
Removing `--dist-dir` is a bit of an awkward change, is there any reason you removed it? I've re-added it locally and everything still seems to work from `/extension`

Reply:
Good question—I was wondering why it was specified in the first place, rather than using the default folder. I couldn’t remember (maybe it was necessary in Parcel v1? or I copied that from someplace?), so I thought it’d be slightly better to leave as much default settings as possible. In this specific example, if I’m new to this codebase a `dist/` folder would look familiar (it’s where built files are stored), but an `extension/` folder can be one of many things.
What do you think?

Reply:
👍🏻 Sounds good. It's better to make it closer to default now, whilst there's minimal dev interruption (ie, two people :P). If we do more work on this it would become more of a pain to switch out